src/unity/toolkits/neural_net/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+project(unity_toolkits)
+
+# The unity_neural_net library provides a platform-agnostic interface to
+# neural-network inference and training backends. It provides a factory method
+# that should instantiate a backend appropriate to the current platform and
+# hardware.
+
+# When building for macOS, build the MetalPerformanceShaders-based backend.
+if(APPLE AND HAS_MPS AND NOT TC_BUILD_IOS)
+    find_library(METAL NAMES Metal)
+    message("Metal found at ${METAL}.")
+
+    find_library(METAL_PERFORMANCE_SHADERS NAMES MetalPerformanceShaders)
+    message("MetalPerformanceShaders found at ${METAL_PERFORMANCE_SHADERS}.")
+
+    make_library(tcmps
```

Comment:
Should this be renamed `neural_net`?

Reply:
Nevermind, looks like we are intentionally keeping a `neural_net` (platform-agnostic) and `tcmps` (macOS-specific) split within this directory.